index.js

```diff
@@ -2,7 +2,19 @@ const assert = require('assert')
 
 module.exports = cacheElement
 
-// Memoize a bel element
-// null -> null
-function cacheElement () {
+function cacheElement (fn) {
+  const store = {}
+  
+  return function render () {
+    const args = Array.from(arguments)
```

Comment:
`Array.from()` isn't quite compatible with most browsers yet; slicing would probably be better - though on a first render there's no need to diff I reckon so this slicing could be avoided if no prior arguments exist